framework/src/org/checkerframework/common/value/statically-executable.astub

```diff
@@ -360,3 +360,39 @@ class Arrays {
     static @MinLen(2) String toString(Object[] a);
     static @MinLen(2) String deepToString(Object[] a);
 }
+
+class Calendar {
+    public @IntRange(from = 1, to = 7) int getFirstDayOfWeek();
+    public void setFirstDayOfWeek(@IntRange(from = 1, to = 7) int value);
+    public @IntRange(from = 1, to = 7) int getMinimalDaysInFirstWeek();
+    public void setMinimalDaysInFirstWeek(@IntRange(from = 1, to = 7) int value) {
```

Comment:
Declaration should end with a `;`.

Reply:
Sorry you had to find this - this was just carelessness on my part. It should be fixed now.